src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpRequestStream.cs

```diff
@@ -110,7 +110,7 @@ public override Task WriteAsync(byte[] buffer, int offset, int count, Cancellati
 
             if (count > buffer.Length - offset)
             {
-                throw new ArgumentException(nameof(buffer));
+                throw new ArgumentException(SR.Argument_BufferNotLongEnough, nameof(buffer));
```

Comment:
Found and fixed.  Bizarrely, it compiles with no errors on my machine (either via Visual Studio or the command line).  Explains why I probably didn't catch it earlier.  I don't know if it's something to do with the solution itself or something about my setup.